store/rootmulti/store.go

```diff
@@ -1162,3 +1131,14 @@ func flushLatestVersion(batch dbm.Batch, version int64) {
 
 	batch.Set([]byte(latestVersionKey), bz)
 }
+
+func flushPruningHeights(batch dbm.Batch, pruneHeights []int64) {
```

Comment:
pretty much the same "setPruningHeights" function, just more inline with the commit info and flush latest version